src/Nowruz/modules/general/components/iconDropDown/useIconDropDown.tsx

```diff
@@ -22,13 +22,20 @@ export const useIconDropDown = () => {
     await nonPermanentStorage.set({ key: 'identity', value: accountId });
     identities()
       .then((resp) => dispatch(setIdentityList(resp)))
-      .then(() => navigate('/jobs'))
+      .then((resp) => {
+        console.log('test log resp', resp);
+        const current = resp.payload.find((item) => item.id === accountId);
+        const path =
```

Comment:
```suggestion
        const type =
          current?.type === 'users'
            ? `users/${(current.meta as UserMeta).username}`
            : `organizations/${(current?.meta as OrgMeta).shortname}`;
            
            navigate(`profile/${type}/view`);
```

Reply:
We are defining the path for navigation based on the type of the user. so I think 'path' is a better option rather than 'type'

Reply:
Yeah, kindly retain `path` variable.